docroot/themes/humsci/humsci_basic/src/js/main-nav/nested-toggler.js

```diff
@@ -0,0 +1,72 @@
+const togglers = document.querySelectorAll('.hb-nested-toggler');
+const mobileNavBreakpoint = 992;
+let windowWidth = window.innerWidth;
+
+if (togglers) {
+  for (let i = 0; i < togglers.length; i += 1) {
+    const toggler = togglers[i];
+    const togglerID = toggler.getAttribute('id');
+    const togglerContent = document.querySelector(`[aria-labelledby="${togglerID}"]`);
+    const togglerParent = toggler.closest('.su-main-nav__item--parent');
```

Comment:
.closest isnt supported in ie11

Reply:
.parentNode might work though!

Reply:
We have Babel to handle this. It looks like there is an error in the secondary nav JS from Decanter that is causing our JS to not run. I'm going to move all Decanter imports to the bottom of the index.js for now so our JS still runs.